AdminTools/Commands/HintBroadcast/User.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using CommandSystem;
+
+namespace AdminTools.Commands.HintBroadcast
+{
+    internal class User
+    {
+    }
+}
```

Comment:
Why we even need it then?

Reply:
Yamato was gonna seperate them
And that's the plan for the future, but currently doing other stuff